### ISSUE TITLE
[ACCESSIBILITY] Orientation Video Play/Pause button needs better Focus indication and to be a Keyboard/Screen reader navigable

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#4873]
**Describe the issue**
There is already a bug for the part where this video should not be automatically playing, ( #4863 ). This bug is to address the focus state of that play/pause button when it is present and the fact that it needs to be navigable by assistive technology like screen readers and keyboard only.

Success Criterion 2.4.7: Focus Visible (Level AA)
Goal: Ensure that users can see which element has keyboard focus.
Requirement: Any keyboard-operable user interface must have a mode where the keyboard focus indicator is visible.
Importance: Without a visible focus indicator, sighted keyboard users cannot effectively navigate the page.

Success Criterion 2.4.13: Focus Appearance (Level AAA)
Goal: Make the keyboard focus indicator more noticeable.
Requirement: The focus indicator must be of sufficient size and contrast. Specifically, it should be at least as large as a 2 CSS pixel thick perimeter of the unfocused component and have a contrast ratio of at least 3:1 between the focused and unfocused states2.

**To Reproduce
Steps to reproduce the behavior:**

Go to newly downloaded app and launch Onboarding screens
Click on 'Continue' until user reaches video screen.

**Expected behavior**
The Screen reader/keyboard only user lands focus on Play/Pause button and there is sufficient color contrast for the user to know what element is receiving focus. 

**Actual behavior**
The video is playing automatically, it should not. Then, when using TalkBack the video opens and plays automatically and the user is unable to understand what TalkBack is saying. This shouldn't play automatically, but if it MUST, then it would be a more accessible experience to have the Screen readers focus land on the Play/Pause "Onboarding video, double tap to activate."  Additionally, as mentioned in Github #4863 "Some users may be very literal and not understand "double tap to activate" as "Play/Pause video" I would suggest changing the label on this to indicate such."  

**Screenshots**

https://github.com/user-attachments/assets/7912b5d8-0b23-404d-bfad-be357c51769a

![Image](https://github.com/user-attachments/assets/955f4f91-7419-44c3-a1bd-e92db9811a10)

**Testing Environment & Devices (please complete the following information):**
Device: Google Pixel 8a
OS: Android 15

**Additional context**

**Comment by acovher2** [comment]
Following the 3/11 meeting with JP and the team, I am updating the way these tickets are categorized to ensure they are placed in the standard bug queue and addressed accordingly. These are legitimate issues that impact users with disabilities, and as critical blockers, they should receive the same level of urgency as other high-priority bugs to ensure equitable access to the application for all users.

**Comment by acovher2** [comment]
@TodorBachvarov This only needs a tiny fix a White border around the play/pause button to provide it sufficient contrast to meet WCAG. The orange over orange is the problematic part here.

All other issues have been opened as unique bugs.
